src/main/java/werkit/Parser.java

```diff
@@ -221,4 +228,26 @@ public ExerciseCommand createExerciseCommand(String userInput) throws
     public HelpCommand createHelpCommand(String userInput) {
         return new HelpCommand(userInput);
     }
+
+    public SearchCommand createSearchCommand(String userInput) throws
+            InvalidCommandException, ArrayIndexOutOfBoundsException {
+        String actionKeyword = userInput.split(" ", 3)[1];
+        String arguments = null;
+        String className = this.getClass().getSimpleName();
+        switch (actionKeyword) {
+        case SEARCH_EXERCISE_ACTION_KEYWORD:
+            if (userInput.split(" ", 3).length < EXPECTED_NUMBER_OF_PARAMETERS_WITH_ARGUMENTS) {
+                logger.log(Level.WARNING, "User has entered an invalid update search exercise command action.");
```

Comment:
Is the word "update" in "update search exercise" supposed to be there?

Reply:
Ohhh, thanks for pointing out, I forgot to remove